doc/cephfs/index.rst

```diff
@@ -42,9 +42,21 @@ Block Devices) and RGW (Ceph Object Storage Gateway).
 Using CephFS
 ============
 
-Using the Ceph File System requires at least one :term:`Ceph Metadata Server` in
-your Ceph Storage Cluster.
+Using CephFS with a running Ceph Storage Cluster requires at least one active
+:doc:`Metadata Server (MDS) daemon </cephfs/add-remove-mds>`, :doc:`creating
+the file system </cephfs/createfs>`, selecting a mount mechanism for clients
+(:doc:`FUSE </cephfs/fuse>` or :doc:`kernel driver </cephfs/kernel>`), and
+configuring :doc:`authentication credentials </cephfs/client-auth>` for
+clients.
 
+For setting up CephFS quickly, checkout the :doc:`CephFS Quick Start document
+</start/quick-cephfs>`.
+
+.. note::
+
+         If cephx authentication is enabled, secret key is required for
+         mounting CephFS with FUSE. It is a file containing key extracted from
+         keyring.
```

Comment:
Maybe link to the cephx doc here?

Also, a few nits:
s/secret key/a secret key/
s/containing key/containing a keyt
s/keyring/the keyring

Reply:
Done